### PR TITLE
AM62XX bootfirmware updates

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

File: meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc [hunk 1]
```diff
@@ -662,7 +662,7 @@ UBOOT_LOADADDRESS:am62xx = "0x82000000"
 UBOOT_DTB_LOADADDRESS:am62xx = "0x88000000"
 UBOOT_DTBO_LOADADDRESS:am62xx = "0x89000000"
 EFI_PROVIDER:am62xx = ""
-LMP_BOOT_FIRMWARE_FILES:am62xx = "tispl.bin u-boot.img tiboot3.bin"
+LMP_BOOT_FIRMWARE_FILES:am62xx = "tispl.bin u-boot.img tiboot3-am62x-gp-evm.bin tiboot3-am62x-hs-evm.bin tiboot3-am62x-hs-fs-evm.bin"
```
Comment (igoropaniuk) [inline meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:665]: Unfortunately it's not possible to rename file using `;`, as it's done for `IMAGE_BOOT_FILES`:
```
IMAGE_BOOT_FILES:append:am62xx-evm = " tiboot3-am62x-gp-evm.bin;tiboot3.bin"
```
So this is why I copy all three for now.

Comment (ricardosalveti) [reply meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:665]: Yeah, this is fine as the usage here is for the boot firmware files recipe anyway.

File: meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging/k3-set-env-variable-device_type.patch [hunk 2]
```diff
@@ -0,0 +1,52 @@
+From 0b638ece826c816a2387834975c9b5854dc94f6b Mon Sep 17 00:00:00 2001
+From: Igor Opaniuk <igor.opaniuk@foundries.io>
+Date: Fri, 21 Jul 2023 18:08:03 +0200
+Subject: [PATCH] k3: set env variable device_type
+
+Set environment variable "device_type", which stores device
+sec state: gp, hs-fs, hs-se.
+
+Upstream-status: not relevant
```
Comment (ricardosalveti) [inline meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging/k3-set-env-variable-device_type.patch:9]: Change to:

`Inappropriate [lmp specific]`

**Review by ricardosalveti — APPROVED** [review]
LGTM, one minor comment, but feel free to merge after updating it.

**Review by MrCry0 — APPROVED** [review]
With "Upstream-status" minor change, LGTM

**Comment by igoropaniuk** [comment]
Comments addressed, thanks!

**Pull request merged**